src/components/transactions/HexEncodedData/index.tsx

```diff
@@ -13,7 +13,10 @@ export const HexEncodedData = ({ hexData, title, limit = 20 }: Props): ReactElem
   const [showTxData, setShowTxData] = useState(false)
   const showExpandBtn = hexData.length > limit
 
-  const toggleExpanded = () => {
+  const toggleExpanded = (
+    e: MouseEvent<HTMLSpanElement, globalThis.MouseEvent> | MouseEvent<HTMLAnchorElement, globalThis.MouseEvent>,
```

Comment:
You don't have to import all that stuff, just do `e: SyntheticEvent`.

Reply:
It's just `MouseEvent` that is imported but as we're not accessing the `target`, I'll adjust it.